packages/dd-trace/src/proxy.js

```diff
@@ -65,6 +68,7 @@ class Tracer extends NoopProxy {
           } else {
             log.debug('Successfully uploaded git metadata')
           }
+          gitMetadataUploadFinishCh.publish(err)
```

Comment:
this seemed like a good use of `diagnostics_channel`. If we think it isn't, there are other alternatives for this